linux/Dockerfile

```diff
@@ -30,7 +30,9 @@ RUN apt-get update && \
 	pkg-config \
 	python2.7 \
 	wget \
-	zip && \
+	zip \
+	nodejs \
```

Comment:
It's a good practice to have packages alphabetically sorted in Dockerfile, so you don't end up with duplicated entries

Reply:
indeed, shall push that change